pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -154,6 +193,36 @@ func (m *manager) Start() {
 	}, 0)
 }
 
+func getCurrentCertificateOrBootstrap(
+	store Store,
+	bootstrapCertificatePEM []byte,
+	bootstrapKeyPEM []byte) (cert *tls.Certificate, rotated bool, errResult error) {
```

Comment:
Previous review feedback, reviewer wanted a name for that boolean so it was more clearly documented.